app/controllers/locations_controller.rb

```diff
@@ -70,7 +70,7 @@ def authenticate_action!
 
   def location_params
     required_params = params.require(:location).permit(:postcode, :name, :total_units, :type_of_unit, :wheelchair_adaptation, :add_another_location).merge(scheme_id: @scheme.id)
-    required_params[:postcode] = required_params[:postcode].delete(" ").upcase.encode("ASCII", "UTF-8", invalid: :replace, undef: :replace, replace: "") if required_params[:postcode]
+    required_params[:postcode] = PostcodeService.clean(required_params[:postcode].delete(" ").upcase) if required_params[:postcode]
```

Comment:
Looks like the `.delete(" ")` already happens in PostcodeService so probably isn't needed here? Maybe the `.upcase` should go there too? I assume we want that everywhere.